testsuite/MDAnalysisTests/analysis/test_hbonds.py

```diff
@@ -31,8 +31,12 @@
 
 import itertools
 import warnings
+try:
```

Comment:
Thank you for your comment. There is no StringIO in py3 so I was trying to make it compatible with py3. I still need some time to figure out how this stream reading works.